src/docnode.cpp

```diff
@@ -2698,7 +2698,157 @@ void DocTitle::parse(DocNodeVariant *thisVariant)
   int tok;
   while ((tok=parser()->tokenizer.lex()))
   {
-    if (!parser()->defaultHandleToken(thisVariant,tok,children()))
+    if (tok == TK_HTMLTAG)
```

Comment:
It looks like there is now quite some overlap/duplication with the `defaultHandleToken()` used in the else, which also handles `tok==TK_HTMLTAG` for a selected set of supported tags. I'd prefer to avoid duplication. Which commands are now handled differently than `defaultHandleToken()`? And are things like `<div>` and `<pre>` really supported as title for all output formats?